packages/flutter_webrtc/tizen/project_def.prop

```diff
@@ -0,0 +1,24 @@
+# See https://docs.tizen.org/application/tizen-studio/native-tools/project-conversion
+# for details.
+
+APPNAME = flutter_webrtc_tizen_plugin
+type = staticLib
```

Comment:
Implicit linking of user libraries (`libwebrtc.so`) is deprecated and will not be supported in the future. Consider changing the project type to `sharedLib` and adding relevant linker options ([example](https://github.com/flutter-tizen/plugins/blob/master/packages/webview_flutter_lwe/tizen/project_def.prop)).

Question: Where's the source code for `libwebrtc.so` and how is it built?

Reply:
I hava changed to shared lib.
The libwebrtc source code uri is https://github.sec.samsung.net/f-project/libwebrtc.